### ISSUE TITLE
Adding populate parameters to the "Populate Where" Blueprint

### ISSUE THREAD

**Issue by iit-federico-bozzini** [balderdashy/sails#3305]
Now it's currently possible to use the Populate Where Blueprint in this way:

    /user/5/books

Sometimes it's necessary to have more information embedded in the element(s) requested through the association (books in this case). It would be great to be able to do something like this:

    /user/5/books?populate=authors&populate=editor

**Comment by rhewitt22** [comment]
[Jason Sims](https://github.com/jasonsims) implemented the above functionality within balderdashy/sails#1695. The only change is that you selectively choose which associations to populate using:
```
localhost:1337/tickets?populate=[]          // Don't populate anything
localhost:1337/tickets?populate=[comments]  // Only populate comments
```
https://stackoverflow.com/questions/23643870/how-to-selectively-populate-waterline-associations-via-query-param-in-sails-js

**Comment by iit-federico-bozzini** [comment]
Unless I missed something, the great work made by Jason Sims doesn't cover my case. A the moment it's possible to populate the associations only when using the find and findOne blueprints.

To get back to my example, it's possibile to do something like:

    users/5?populate=books

It doesn't work with the Populate Blueprint.

**Comment by sailsbot** [comment]
Thanks for posting, @iit-federico-bozzini.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
